mulesoft-inference/1.1/modules/ROOT/pages/mulesoft-inference-connector-reference.adoc

```diff
@@ -0,0 +1,2489 @@
+| Algorithm a| String | Encryption algorithm that the truststore uses. |  |
+| Insecure a| Boolean | If `true`, Mule stops performing certificate validations. Setting this to `true` can make connections vulnerable to attacks. |  |
+|===
+
+[[KeyStore]]
+=== Keystore
+
+Configures the keystore for the TLS protocol. The keystore you generate contains a private key and a public certificate.
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Path a| String | Path to the keystore. Mule resolves the path relative to the current classpath and file system. |  |
+| Type a| String | Type of keystore. |  |
+| Alias a| String | Alias of the key to use when the keystore contains multiple private keys. By default, Mule uses the first key in the file. |  |
+| Key Password a| String | Password used to protect the private key. |  |
+| Password a| String | Password used to protect the keystore. |  |
+| Algorithm a| String | Encryption algorithm that the keystore uses. |  |
+|===
+
+[[StandardRevocationCheck]]
+=== Standard Revocation Check
+
+Configures standard revocation checks for TLS certificates.
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Only End Entities a| Boolean a| Which elements to verify in the certificate chain:
+
+* `true`
++
+Verify only the last element in the certificate chain.
+
+* `false`
++
+Verify all elements in the certificate chain. |  |
+| Prefer Crls a| Boolean a| How to check certificate validity:
+
+* `true`
++
+Check the Certification Revocation List (CRL) for certificate validity.
+
+* `false`
++
+Use the Online Certificate Status Protocol (OCSP) to check certificate validity. |  |
+| No Fallback a| Boolean a| Whether to use the secondary method to check certificate validity:
+
+* `true`
++
+Use the method that wasn't specified in the *Prefer Crls* field (the secondary method) to check certificate validity.
+
+* `false`
++
+Do not use the secondary method to check certificate validity. |  |
+| Soft Fail a| Boolean a| What to do if the revocation server can't be reached or is busy:
+
+* `true`
++
+Avoid verification failure.
+
+* `false`
++
+Allow the verification to fail. |  |
+|===
+
+[[CustomOcspResponder]]
+=== Custom OCSP Responder
+
+Configures a custom OCSP responder for certification revocation checks.
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Url a| String | URL of the OCSP responder. |  |
+| Cert Alias a| String | Alias of the signing certificate for the OCSP response. If specified, the alias must be in the truststore. |  |
+|===
+
+
+[[CrlFile]]
+=== CRL File
+
+Specifies the location of the certification revocation list (CRL) file.
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Path a| String | Path to the CRL file. |  |
+|===
+[[DefaultHeader]]
```

Comment:
add a line gap between line 2268 and 2269, and remove the line gap between 2269 and 2270